azafea/event_processors/endless/metrics/v2/migrations/e7c7d5f2e4df_materialized_view_for_metrics.py

```diff
@@ -0,0 +1,27 @@
+# type: ignore
+
+"""Materialized view for metrics.
+
+Revision ID: e7c7d5f2e4df
+Revises: d4809698244a
+Create Date: 2020-06-09 13:01:05.739571
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'e7c7d5f2e4df'
+down_revision = 'd4809698244a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute(
+        "CREATE MATERIALIZED VIEW metrics_request_v2_by_day (month, machine_id) AS "
```

Comment:
Shouldn't `month` be called `date` or am I missing something?

Reply:
Of course…